backend/lib/server_code/api/roomRoutes.js

```diff
@@ -1,11 +1,16 @@
 const express = require("express");
-const { socket } = require("./socket.js");
+const { createNewSocket } = require("./socket.js");
+
+const socketsCollection = {};
```

Comment:
isn't it supposed to be room2players?
like:
room ids: '1', '2'.
players ids: '111', '222', '333', '444'
{1:[111, 333], 2: [222, 444]}

if so, the naming should be room2playersSockets

Reply:
talked about it